src/as2fm/jani_generator/jani_entries/jani_edge.py

```diff
@@ -18,7 +18,7 @@
 from typing import Dict, Optional
 
 from as2fm.jani_generator.jani_entries import (JaniAssignment, JaniConstant,
```

Comment:
Can we enforce going to a new line as soon as `import (...` is found?

Reply:
somehow black has the philosophy of non-configurability. idk.